### PR TITLE
Cast openshift_enable_origin_repo to bool.

### LINKED ISSUE #2972: var 'openshift_enable_origin_repo' can not be overwrite through host file
It seems that we can set 'true' or 'false' to var 'openshift_enable_origin_repo' to decide if copy repo file 'openshift-ansible-centos-paas-sig.repo' to target machine. But actually it doesn't work for me. 

##### Version
ansible version 2.0

##### Steps To Reproduce
1. set openshift_enable_origin_repo=false to hosts file
2. install openshift through openshift-ansible

##### Current Result
openshift repo file copy to target machine

##### Expected Result
openshift repo file don't copy to target machine

I'm not sure if it is a bug. refer source code 'openshift-ansible\roles\openshift_repos\tasks\main.yml'

below task 'Configure origin yum repositories RHEL/CentOS'

```
- name: Configure origin yum repositories RHEL/CentOS
  copy:
    src: origin/repos/openshift-ansible-centos-paas-sig.repo
    dest: /etc/yum.repos.d/
  notify: refresh cache
  when: ansible_os_family == "RedHat" and ansible_distribution != "Fedora"
        and openshift_deployment_type == 'origin'
        and not openshift.common.is_containerized | bool
        and openshift_enable_origin_repo | default(true)
```
**should be updated by**
```
- name: Configure origin yum repositories RHEL/CentOS
  copy:
    src: origin/repos/openshift-ansible-centos-paas-sig.repo
    dest: /etc/yum.repos.d/
  notify: refresh cache
  when: ansible_os_family == "RedHat" and ansible_distribution != "Fedora"
        and openshift_deployment_type == 'origin'
        and not openshift.common.is_containerized | bool
        and openshift_enable_origin_repo **| bool** | default(true)
```

### PR DESCRIPTION
**Pull request by abutcher**
Fixes #2972

### CODE REVIEW

**Comment by abutcher** [comment]
aos-ci-test

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
aos-ci-test

**Comment by sdodson** [comment]
aos-ci-test

File: roles/openshift_repos/tasks/main.yaml [hunk 1]
```diff
@@ -47,4 +47,4 @@
   when: ansible_os_family == "RedHat" and ansible_distribution != "Fedora"
         and openshift_deployment_type == 'origin'
         and not openshift.common.is_containerized | bool
-        and openshift_enable_origin_repo | default(true)
+        and (openshift_enable_origin_repo | bool) | default(true)
```
Comment (detiber) [inline roles/openshift_repos/tasks/main.yaml]: Would it make sense to apply the default filter before the bool filter?

**Comment by abutcher** [comment]
aos-ci-test

**Pull request merged**